app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,44 @@
+require 'uri'
+require 'net/http'
+require 'json'
+
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(benchmark_run_id, benchmark_type_id, benchmark_result_type_id)
+
+    benchmark_run = BenchmarkRun.find_by_id(benchmark_run_id)
+    benchmark_type = BenchmarkType.find_by_id(benchmark_type_id)
+    benchmark_result_type = BenchmarkResultType.find_by_id(benchmark_result_type_id)
+
+    category = benchmark_run.result.keys[0]
+    current_id = benchmark_run_id
+    last_id = BenchmarkRun.where(initiator_type: Commit, benchmark_type: benchmark_type,
+                     benchmark_result_type: benchmark_result_type).order(:created_at).first.id
+
+    previous_benchmark_runs = BenchmarkRun.where("id not in (?)",(current_id..last_id)).where(initiator_type: Commit, benchmark_type: benchmark_type,
```

Comment:
Previously, I was doing with benchmark_ids, which I realized was incorrect. Now, I just remove the ids which came after my benchmark_run so that if another benchmark runs before this job is executed it won't be a problem. How do I achieve this by ordered by created at?